guides/2020-11-09-installing-go-programs-directly/en.markdown

```diff
@@ -14,7 +14,7 @@ can include at the top of their `README.md` explaining how to install the tool.
 
 Go 1.16 introduces a new way to install Go programs directly with the `go` command. This guide
 introduces the new mode of `{{{ .cmdgo.install }}}` using the example of
-[`{{{ .mkcert_pkg }}}`](https://mkcert.io/), and is suitable for both end users and tool authors.
+[`{{{ .mkcert_pkg }}}`](https://github.com/FiloSottile/mkcert), and is suitable for both end users and tool authors.
```

Comment:
Good catch, thank you!

My suggestion here would be that we change this to:

```
[`{{{ .mkcert_pkg }}}`](https://{{{ .mkcert_pkg }}})
```

because that way we do not hardcode the _current_ `go-import` target of `filippo.io/mkcert` (it could change if Filippo chooses to host the module elsewhere).